monai/apps/pathology/transforms/post/array.py

```diff
@@ -624,3 +626,98 @@ def __call__(  # type: ignore
         type_prob = type_dict[inst_type] / (sum(seg_map_crop) + 1.0e-6)
 
         return (int(inst_type), float(type_prob))
+
+
+class HoVerNetPostProcessing(Transform):
+    """
+    Since HoVerNet do segmentation and classification meanwhile, this transform is used to combine postprocessing
+    for segmentation and classification. It assumes input as a dictionary.
+
+    Args:
+        seg_postpprocessing: execute post-processing transformation for the segmentation output from model.
+            Typically, several Tensor based transforms composed by `Compose`.
+        distance_map_key: the key pointing to the distance map generated by `seg_postprocessing`.
+        points_num: assumed that the created contour does not form a contour if it does not contain more points
+            than the specified value. Defaults to 3.
+        level: optional. Used in `skimage.measure.find_contours`. Value along which to find contours in the array.
+            By default, the level is set to (max(image) + min(image)) / 2.
+        return_binary: whether to return the binary segmentation prediction after `seg_postprocessing`.
+        pred_binary_key: if `return_binary` is True, this `pred_binary_key` is used to get the binary prediciton
+            from the output.
+        return_centroids: whether to return centroids for each instance.
+        output_classes: number of the nuclear type classes.
+        inst_info_dict_keys: key use to record information for each instance.
+    """
+
+    def __init__(
+        self,
+        seg_postpprocessing: Callable,
+        distance_map_key: str = "dist",
+        points_num: int = 3,
```

Comment:
Isn't it clearer to name it `min_num_points`. Also inside `GenerateInstanceContour`, which we missed in the previous PR. Would be great if you can fix it. Thanks